core/async_job/async_job.ex

```diff
@@ -23,6 +24,7 @@ defmodule AntikytheraCore.AsyncJob do
     remaining_attempts: Attempts,
     retry_interval:     RetryInterval,
     payload:            Croma.Map, # opaque data given and used by gear
+    immediate:          Croma.Boolean,
```

Comment:
Adding this field to `AntikytheraCore.AsyncJob` struct is rather tricky.
Instances of the current version of this struct (i.e. without `:immediate` field) already exist in the replicated Raft consensus groups and they have to be properly migrated to the new version (i.e. with `:immediate field).
Can we just omit this `:immediate` field?

Reply:
Yes, the field is not necessary.

Fixed in 0a17fb1cc0e34322e8281898a5ec061cef1ecb01